src/Adapter/Image/ImageRetriever.php

```diff
@@ -205,7 +205,7 @@ public function getImage($object, $id_image)
             $url = $this->link->$getImageURL(
                 isset($object->link_rewrite) ? $object->link_rewrite : $object->name,
                 $id_image,
-                $image_type['name']
+                $image_type['name'] . ($generateHighDpiImages ? '2x' : '')
```

Comment:
seems like this `2x` string is used somewhere else in this file, can we put it in a constant instead of hardcoding it two times?

Reply:
Thanks for the feedback. Do you prefer:
- a `define`?
- a `const` in the class declaration?
- a variable declared in the function?

Reply:
a `const` in the class declaration

Reply:
Done